src/app/clusters/pressure-measurement-cluster.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
```

Comment:
This should presumably live in some directory where ZAP will find it, which will fix the "this is removing this cluster from everything" problem.  Something like `src/app/zap-templates/zcl/data-model/chip` perhaps?